### ISSUE TITLE
How would I retrieve a frequency value?

### ISSUE THREAD

**Issue by Lunathecatte** [filoe/cscore#407]
Like a single value of a live audio device. lets say a speaker? I have never worked with CSCore and hope that it indeed is very much possible with it

I am looking forward to youre replies!

**Comment by Lunathecatte** [comment]
so that would mean like something like an int coming out of it so i can process that easily

**Comment by filoe** [comment]
Frequency of what? An audio signal usually consists out of multiple signals, with different frequencies, interfering with each other. So there is usually not one "frequency". Also there is no frequency at specific point of time. You have to take a look at a certain time range. 

Maybe you want to calc a fft and determine the major frequency?

**Comment by Lunathecatte** [comment]
Yes honestly, that is what i want. My english is just not great. Im sorry

**Comment by LeonardVermeer** [comment]
@filoe
Is there a way to extract all of those frequencies of a song individually over the span of let's say a second? I imagine it could be something in the lines of 20Hz-60Hz @10 dB and 61Hz-250Hz @ 3 dB and so on for that second...
Or is it possible to get the superimposed frequencies over the span of a second?
Would something like that be possible with c#?
